models/reviews.model.js

```diff
@@ -1,5 +1,35 @@
 const db = require("../db/connection.js")
 
+exports.fetchReviewsQuery = (params) => {
+    let sql = ` SELECT * FROM reviews`
+
+    if(params.category) sql += ` WHERE category = '${params.category}'`
+
+    if(params.sort_by){
+        const column = params.sort_by;
+        if(column === 'review_id') sql += ` ORDER BY ${column}`
```

Comment:
Instead of writing the same code this many times, try to store those categories, then check them once.